app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -140,6 +141,11 @@ protected void onCreate(final Bundle savedInstanceState) {
         ThemeHelper.setDayNightMode(this);
         ThemeHelper.setTheme(this, ServiceHelper.getSelectedServiceId(this));
 
+        // Fixes text color turning black in dark/black mode:
+        // https://github.com/TeamNewPipe/NewPipe/issues/12016
+        // For further reference see: https://issuetracker.google.com/issues/37124582
+        new WebView(this);
```

Comment:
Please ensure there is a WebView provider available with [`DeviceUtils.supportsWebView`](https://github.com/TeamNewPipe/NewPipe/blob/c6e17218843e59f264f39d9cd719c1173a95abfd/app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java#L344), otherwise users without a WebView wouldn't be able to use the app, as it would crash due to no WebView installed or enabled (already happening for licenses' WebViews)

Reply:
Oh, right, nice catch. You could also add a try-catch around it, just to make sure.

Reply:
Oh yes I forgot. Then no need for an additional try-catch, *nice catch* ;-)

Reply:
Actually I meant to add a try-catch around the webview instantiation, which might still fail for unknown reasons